docs/developer/process/integration-release.md

```diff
@@ -225,6 +228,31 @@ securesystemslib.exceptions.FormatError: '[none]' did not match 'pattern /[a-fA-
         deleted tags, so any subsequent manual trigger tags will need to increment the version number.
 
     1. Delete the branch and tag, locally and on GitHub.
+    
+- If the [build pipeline](../meta/cd.md) failed due to another feature PR being merged after the release PR is opened, the merged release PR will not have updated and signed the feature PR's files.
+  
```

Comment:
Aren't the 2 `Build pipeline failed` troubleshooting steps the same?

Reply:
the first one is sign only and the second is creating a manual version bump

Reply:
Can we make it more explicit?